CHANGELOG.md

```diff
@@ -26,7 +26,7 @@
 
 ## Fixed Issues
 
--
+- [core] Fix type error to allow filtering on one-to-many navigation properties.
```

Comment:
Minor: It would be great if you can mention lambda expression (is typed as one to many navigation properties in core), as the author of the issue failed to use lambda expression.